packages/@lwc/integration-karma/test/mixed-shadow-mode/LightningElement.shadowSupportMode/index.spec.js

```diff
@@ -40,3 +41,26 @@ describe('ENABLE_MIXED_SHADOW_MODE', () => {
         setFeatureFlagForTest('ENABLE_MIXED_SHADOW_MODE', false);
     });
 });
+
+describe('ENABLE_NATIVE_SHADOW_MODE', () => {
+    beforeEach(() => {
+        setFeatureFlagForTest('ENABLE_MIXED_SHADOW_MODE', true);
+    });
```

Comment:
Native shadow should be used without this feature flag so we don't need to enable here.